groups/bsl/bslstl/bslstl_variant.h

```diff
@@ -951,8 +995,13 @@ struct Variant_isAlternativeConstructibleFrom
       Variant_IsConstructible<
           typename bsl::variant_alternative<INDEX, VARIANT>::type,
           TYPE>::value &&
-          Variant_HasUniqueCVType<BSLSTL_VARIANT_CONVERT_TYPE_OF(TYPE, VARIANT),
-                                VARIANT>::value> {
+          Variant_HasUniqueCVType<BSLSTL_VARIANT_CONVERT_TYPE_OF(TYPE,
+                                                                 VARIANT),
+                                  VARIANT>::value> {
+    // This component-private metafunction is derived from 'bsl::true_type' if
+    // there is a unique best match alternative in the specified template
+    // parameter 'VARIANT' that be constructed from the specified template
```

Comment:
```suggestion
    // parameter 'VARIANT' for construction from the specified template
```
This rewording deliberately does not imply that the selected alternative actually *can* be constructed from `TYPE`, since that check is done later, in `Variant_ConstructsFromType`.